nautobot/extras/models/jobs.py

```diff
@@ -835,10 +838,10 @@ class ScheduledJob(BaseModel):
         help_text='The name of the Celery task that should be run. (Example: "proj.tasks.import_contacts")',
         db_index=True,
     )
-    # Note that we allow job_model to be null and use models.SET_NULL here.
+    # Note that we allow job to be null and use models.SET_NULL here.
     # This is because we want to be able to keep ScheduledJob records for tracking and auditing purposes even after
     # deleting the corresponding Job record.
-    job_model = models.ForeignKey(
+    job = models.ForeignKey(
```

Comment:
I'm not 100% on board with this change. We currently have `job_model` and `job_class` to clearly indicate which is which; IMO changing this to simply `job` makes things a bit ambiguous.

Reply:
I think `job` is a more intuitive and straightforward approach, as it directly represents a `job` instance. On the other hand, the use of `job_model` and `job_class` could potentially lead to confusion, since in Django a model is as a class. This could cause the terms `job_model` and 'job_class' to be perceived as synonymous, potentially leading to misunderstandings. IMO